components/cli/pkg/commands/setup_create_local.go

```diff
@@ -54,58 +40,60 @@ func createLocal() error {
 	defer func() {
 		spinner.Stop(true)
 	}()
-
-	if !addGlobalGateway && !addObservability {
-		createBasicLocalEnvironment(vmLocation)
-	}
-
-	if addGlobalGateway && !addObservability {
-		createLocalWithGlobalGateway(vmLocation)
-	}
-
-	if addGlobalGateway && addObservability {
-		createLocalEnvironmentWithObservability(vmLocation)
+	if isCompleteSelected {
+		vmName = constants.AWS_S3_ITEM_VM_COMPLETE
+	} else {
+		vmName = constants.AWS_S3_ITEM_VM_MINIMAL
 	}
+	util.DownloadFromS3Bucket(constants.AWS_S3_BUCKET, vmName, vmLocation)
+	util.ExtractTarGzFile(vmLocation, filepath.Join(util.UserHomeDir(), constants.CELLERY_HOME, constants.VM, vmName))
 	util.DownloadFromS3Bucket(constants.AWS_S3_BUCKET, constants.AWS_S3_ITEM_CONFIG, vmLocation)
 	util.ReplaceFile(filepath.Join(util.UserHomeDir(), ".kube", "config"), filepath.Join(util.UserHomeDir(), constants.CELLERY_HOME, constants.VM, constants.AWS_S3_ITEM_CONFIG))
-	installVM()
-
-	return nil
-}
-
-func createBasicLocalEnvironment(vmLocation string) error {
-	util.DownloadFromS3Bucket(constants.AWS_S3_BUCKET, constants.AWS_S3_ITEM_VM, vmLocation)
-	util.ExtractTarGzFile(vmLocation, filepath.Join(util.UserHomeDir(), constants.CELLERY_HOME, constants.VM, constants.AWS_S3_ITEM_VM))
-
-	return nil
+	installVM(isCompleteSelected)
 }
 
-func createLocalWithGlobalGateway(vmLocation string) error {
-	util.DownloadFromS3Bucket(constants.AWS_S3_BUCKET, constants.AWS_S3_ITEM_VM, vmLocation)
-	util.ExtractTarGzFile(vmLocation, filepath.Join(util.UserHomeDir(), constants.CELLERY_HOME, constants.VM, constants.AWS_S3_ITEM_VM))
-
-	return nil
-}
+func createLocal() error {
+	var isCompleteSelected = false
+	cellTemplate := &promptui.SelectTemplates{
+		Label:    "{{ . }}",
+		Active:   "\U000027A4 {{ .| bold }}",
+		Inactive: "  {{ . | faint }}",
+		Help:     util.Faint("[Use arrow keys]"),
+	}
 
-func createLocalEnvironmentWithObservability(vmLocation string) error {
-	util.DownloadFromS3Bucket(constants.AWS_S3_BUCKET, constants.AWS_S3_ITEM_VM, vmLocation)
-	util.ExtractTarGzFile(vmLocation, filepath.Join(util.UserHomeDir(), constants.CELLERY_HOME, constants.VM, constants.AWS_S3_ITEM_VM))
+	cellPrompt := promptui.Select{
+		Label:     util.YellowBold("?") + " Select the type of runtime",
+		Items:     []string{"Basic", "Complete"},
+		Templates: cellTemplate,
+	}
+	_, value, err := cellPrompt.Run()
+	if err != nil {
+		return fmt.Errorf("Failed to select an option: %v", err)
+	}
+	if value == "Basic" {
+		isCompleteSelected = true
+	}
+	RunSetupCreateLocal(isCompleteSelected)
 
 	return nil
 }
 
-func installVM() error {
+func installVM(isCompleteSelected bool) error {
+	vmName := constants.VM_NAME
+	if isCompleteSelected {
+		vmName = constants.VM_NAME
```

Comment:
if condition is not required here?

Reply:
Updated constant name to a different one.